### PR TITLE
Streamline away Joss paper verbosity

### PR DESCRIPTION
**Pull request by mmore500**
Currently,  3,346 words 24,459 characters   -> 2,755 words 20,842 characters (~20% cut :axe:)

### CODE REVIEW

**Comment by mmore500** [comment]
We're a smidge over five pages for the main content --- entering the realm of mostly reasonable! [vjv6irkjbthulm2mczvg.pdf](https://github.com/devosoft/Empirical/files/11699584/vjv6irkjbthulm2mczvg.pdf)

File: paper.md [hunk 1]
```diff
@@ -73,16 +73,16 @@ JOSS: "Begin your paper with a summary of the high-level functionality of your s
 i.e., What does this software do?
 -->
 
-Empirical is a C++ library designed to promote open science and facilitate the development of scientific software that is efficient, reliable, and broadly available.
-Specifically, we intend for both the library and the software built with it to meet the following goals:
+Empirical is a C++ library designed to promote open science and facilitate the development of scientific software that is efficient, reliable, and universal distributability to researchers and non-experts alike.
```
Comment (amlalejini) [inline paper.md:76]: `universal distributability to researchers and non-experts alike` Something's off here, making it a little confusing. Maybe `universal distributability` => `easily distributed`?

File: paper.md [hunk 2]
```diff
@@ -73,16 +73,16 @@ JOSS: "Begin your paper with a summary of the high-level functionality of your s
...
-4. **Availability:** Scientific software is most useful when it can be easily accessed by any researcher, student, or citizen scientist who wants to experiment, explore, or test the reproducibility of results.  As such, Empirical is highly portable, uses common data formats, flexible and intuitive configuration options, and tools to turn your software into a performant web app, without requiring a deep understanding of web development.
+1. **Utility:** Empirical features a wide selection of helper tools to streamline common scientific computing tasks such as configuration, data management, random number generation, mathematical manipulations, and signal-action registration. Examples, documentation, and project templates streamline developer orientation.
+2. **Efficiency:** Empirical tools emphasize efficiency to make make heavy scientific workloads tractable. Where possible, computation is moved to compile-time, heap allocation is avoided, and memory-locality is respected.
```
Comment (amlalejini) [inline paper.md:80]: `make make` => `make`

File: paper.md [hunk 3]
```diff
@@ -332,34 +328,36 @@ To this end, we look forward to supporting a wider of collection of collaborator
 <!-- JOSS: Do the authors describe how this software compares to other commonly-used packages?-->
 
 
-There are many existing software platforms that provide support for the different functionalities that Empirical also provides.
-However, most are not in C++, and there is value in this functionality being easily available to programers .
+There are many existing software platforms that provide functionalities overlapping with Empirical.
+However, most are not in C++, and there is value in this functionality being easily available to programmers.
```
Comment (amlalejini) [inline paper.md:332]: C++ programmers (instead of just `programmers`)? Or is the more general term intentional?

**Review by amlalejini — COMMENTED** [review]
There are a few places with potentially unintended paragraph breaks. Not strictly necessary to merge this in, but prior to submission, we should make sure that everything is as intended when rendered. 

I think we can also start stripping out many of the comments we made while drafting (not necessary for this merge, but probably should be done prior to submission).

File: paper.md (hunk 2 above)
Comment (mmore500) [reply paper.md]: :see_no_evil:

**Review by amlalejini — APPROVED** [review]
Looks good! I support merging this into the main joss-paper branch.

**Pull request merged**